inst/include/Rcpp/api/meat/Rcpp_eval.h

```diff
@@ -54,7 +54,7 @@ inline SEXP Rcpp_eval(SEXP expr, SEXP env) {
     SET_TAG(CDDR(CDR(call)), ::Rf_install("interrupt"));
 
     // execute the call
-    Shield<SEXP> res(::Rf_eval(call, R_GlobalEnv));
+    Shield<SEXP> res(::Rf_eval(call, env));
```

Comment:
Why the switch from `R_GlobalEnv` to `env` here but not on [line 69](https://github.com/thirdwing/Rcpp/blob/20d02de09b155b10cb463b7a3adea35dd8267fab/inst/include/Rcpp/api/meat/Rcpp_eval.h)? 

```cpp
Shield<SEXP> conditionMessage(::Rf_eval(conditionMessageCall, R_GlobalEnv));
```